lisa/tools/qemu.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from assertpy.assertpy import assert_that
+
+from lisa.executable import Tool
+from lisa.operating_system import Fedora, Posix
+from lisa.tools import Lsmod
+
+
+class Qemu(Tool):
+
+    QEMU_INSTALL_LOCATIONS = ["qemu-system-x86_64", "qemu-kvm", "/usr/libexec/qemu-kvm"]
+    COMMAND = "qemu-system-x86_64"
```

Comment:
It looks used as an instance variable, not a constant or class level variable. So not to define it to class level. The class level variable will share among classes. If we run this test cases on different distros together, it may have problems.

Reply:
This needs to be defined in class level variable because we can't create an init method to define a instance specific variable. This is because  instance specific variable needs to be defined in init and init method requires Node object which creates a circular import. But from what I understand about global variables is that a copy is maintained for each class.

Reply:
1. In Python, it doesn't need to declare a variable in `__init__`. It can be created anytime, but just make sure it's used after it's set.
2. In the tool, use `_initialize` to initialize for an object.
3. If there is circular import error on declaration. Try 1) use `if TYPING_CHECKING:` to limit the import statement in typing check phase only. 2) quote it like `def method_name(self, node: "Node") -> None:`.

Reply:
_initialize seems to work. Thanks!